playbooks/beegfs.yml

```diff
@@ -3,24 +3,24 @@
 # for a BeeGFS server 
 
 - hosts:
-    -  beegfs_servers
-    -  beegfs_clients
+    -  beegfs_servers:&cluster
+    -  beegfs_clients:&cluster
   roles:
     # Note: The BeeGFS clients are likely to be drawn from across a multi-stack
     # platform deployment. This playbook assumes the existence of a meta-group,
     # beegfs_clients, which contains all clients in the cluster.
     - role: stackhpc.beegfs
       beegfs_enable:
         admon: false
-        mgmt: "{{ inventory_hostname in groups[beegfs_name + '_mds'] }}"
-        meta: "{{ inventory_hostname in groups[beegfs_name + '_mds'] }}"
-        oss: "{{ inventory_hostname in groups[beegfs_name + '_oss'] }}"
-        tuning: "{{ inventory_hostname in groups[beegfs_name + '_oss'] }}"
+        mgmt: "{{ 'mds' in groups and inventory_hostname in groups['mds'] }}"
```

Comment:
should be `'mgmt' in groups and  inventory_hostname in groups['mgmt']`

Reply:
We don't create any `mgmt` group, see https://github.com/cedadev/jasmin-appliances/blob/slurm-beegfs/group_vars/openstack/beegfs#L15-L17

Reply:
Ah makes sense